publication-commons/src/main/java/no/unit/nva/publication/service/impl/ResourceService.java

```diff
@@ -85,10 +89,14 @@ public ResourceService(AmazonDynamoDB client, Clock clock, Supplier<SortableIden
         this.readResourceService = new ReadResourceService(client, RESOURCES_TABLE_NAME);
         this.updateResourceService =
             new UpdateResourceService(client, RESOURCES_TABLE_NAME, clockForTimestamps, readResourceService);
+        var affiliationService = AffiliationSelectionService.create(externalServicesHttpClient);
+        attempt(() -> affiliationService.fetchAffiliation("make PMD happy")).orElseThrow();
```

Comment:
This line will be removed on the next PR. Added only to make it possible to include the affiliationService without actually using it and making PMD happy.